api/src/test/java/org/openmrs/api/ProgramWorkflowServiceTest.java

```diff
@@ -1095,6 +1101,42 @@ public void triggerStateConversion_shouldTestTransitionToState(){
 		pwsi.triggerStateConversion(patient, trigger, dateConverted);
 		assertEquals(patientProgram.getStates().size(), (patientStatesSize + 1));
 	}
+
+	/**
+	 * Test to check the implementation of Patient Program Attribute Type by Uuid
+	 * from Program Workflow Service
+	 * @see ProgramWorkflowService#getPatientProgramAttributeByUuid(String) 
+	 */
+	@Test
+	public void getProgramAttributeType_shouldPass(){
+
+		PatientProgramAttribute patientProgramAttribute = pws.getPatientProgramAttributeByUuid("dd8793dc-e9d2-11ee-9b67-333dbdea27a2");
+		assertNotNull(patientProgramAttribute);
+		assertEquals(2, patientProgramAttribute.getId());
+	}
+
+	/**
+	 * Test to check the number of Patient Program attributes by given name
+	 * @see ProgramWorkflowService#getPatientProgramAttributeByAttributeName(List, String) 
+	 */
+	@Test
+	public void getProgramAttributeByAttributeName_shouldPass(){
```

Comment:
Instead of `shouldPass` do you mind taking a look at the rest of the tests to follow the same naming convention? 😊

Reply:
Sure